core/index-api.js

```diff
@@ -319,12 +319,9 @@ let findTx = async (ctx) => {
 			fee: (tx.fees || 0) + '',
 			hash: tx.hash,
 			block: tx.block || undefined,
-			error: tx.error || undefined
+			error: tx.error || undefined,
+			errorCode: tx.errorCode
```

Comment:
don't we need tx.errorCode || "unknown" here?

Reply:
Not really - it's nullable in contract.